include/seqan3/io/alignment_file/record.hpp

```diff
@@ -0,0 +1,415 @@
+    {
+        return std::get<0>(get_impl(field_constant<seqan3::field::mate>{}, static_cast<tuple_base_t const &>(*this)));
+    }
+
+    //!\brief The mate pair information with the position.
+    decltype(auto) mate_position() &&
+    {
+        return std::get<1>(get_impl(field_constant<seqan3::field::mate>{}, static_cast<tuple_base_t &&>(*this)));
+    }
+    //!\copydoc seqan3::sam_record::mate_position
+    decltype(auto) mate_position() const &&
+    {
+        return std::get<1>(get_impl(field_constant<seqan3::field::mate>{}, static_cast<tuple_base_t const &&>(*this)));
+    }
+    //!\copydoc seqan3::sam_record::mate_position
+    decltype(auto) mate_position() &
+    {
+        return std::get<1>(get_impl(field_constant<seqan3::field::mate>{}, static_cast<tuple_base_t &>(*this)));
+    }
+    //!\copydoc seqan3::sam_record::mate_position
+    decltype(auto) mate_position() const &
+    {
+        return std::get<1>(get_impl(field_constant<seqan3::field::mate>{}, static_cast<tuple_base_t const &>(*this)));
+    }
+
+    //!\brief The mate pair information with the template length.
+    decltype(auto) template_length() &&
+    {
+        return std::get<2>(get_impl(field_constant<seqan3::field::mate>{}, static_cast<tuple_base_t &&>(*this)));
+    }
+    //!\copydoc seqan3::sam_record::template_length
+    decltype(auto) template_length() const &&
+    {
+        return std::get<2>(get_impl(field_constant<seqan3::field::mate>{}, static_cast<tuple_base_t const &&>(*this)));
+    }
+    //!\copydoc seqan3::sam_record::template_length
+    decltype(auto) template_length() &
+    {
+        return std::get<2>(get_impl(field_constant<seqan3::field::mate>{}, static_cast<tuple_base_t &>(*this)));
+    }
+    //!\copydoc seqan3::sam_record::template_length
+    decltype(auto) template_length() const &
+    {
+        return std::get<2>(get_impl(field_constant<seqan3::field::mate>{}, static_cast<tuple_base_t const &>(*this)));
+    }
+
+    //!\brief The mapping quality of the seqan3::field::seq alignment, usually a Phred-scaled score. 
+    decltype(auto) mapping_quality() &&
+    {
+        return get_impl(field_constant<seqan3::field::mapq>{}, static_cast<tuple_base_t &&>(*this));
+    }
+    //!\copydoc seqan3::sam_record::mapping_quality
+    decltype(auto) mapping_quality() const &&
+    {
+        return get_impl(field_constant<seqan3::field::mapq>{}, static_cast<tuple_base_t const &&>(*this));
+    }
+    //!\copydoc seqan3::sam_record::mapping_quality
+    decltype(auto) mapping_quality() &
+    {
+        return get_impl(field_constant<seqan3::field::mapq>{}, static_cast<tuple_base_t &>(*this));
+    }
+    //!\copydoc seqan3::sam_record::mapping_quality
+    decltype(auto) mapping_quality() const &
+    {
+        return get_impl(field_constant<seqan3::field::mapq>{}, static_cast<tuple_base_t const &>(*this));
+    }
+
+    //!\brief The cigar vector (std::vector<seqan3::cigar>) representing the alignment in SAM/BAM format.
```

Comment:
```suggestion
    //!\brief The cigar vector (std::vector<seqan3::cigar>) representing the alignment.
```

There's nothing else now :)